.releaserc

```diff
@@ -6,9 +6,17 @@
   "repositoryUrl": "git@github.com:forkbombeu/ncr.git",
   "dryRun": false,
   "plugins": [
-    "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
     "@semantic-release/changelog",
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "releaseRules": [
+          { "type": "build", "scope": "pnpm-lock", "release": "patch" }
```

Comment:
I dunno... It's something like:
`build(pnpm-lock): zenroom update`
correct?

Reply:
MAsssssI VA BENE

Reply:
yes it is like that

Reply:
maybe more appropriate chore(pnpm-lock)?